.gitignore

```diff
@@ -1,5 +1,5 @@
 # Project-specific files
```

Comment:
Once we have curated data (hand-selected topics for pieces of feedback) I can set up tests for ranges of correctness.